### DISCUSSION TITLE
hasmany RelationManager not shown

### DISCUSSION THREAD

**Discussion by ishaarany** [filamentphp/filament#2507] [Help]
### Package

filament/filament

### Package Version

V2.10

### Laravel Version

v9.2

### Livewire Version

v2..0.0

### PHP Version

PHP 8.0.2

### Bug description

I have TeamResource which have hasmany RelationManager(PrivilagesRelationManager), I register the PrivilagesRelationManager at TeamResource's getRelations() method, but nothing is shown.

### Steps to reproduce

1- create has many relation at Team model
![Team Model](https://user-images.githubusercontent.com/16566638/169678148-0c780c37-f02c-4c3d-9b4d-0cf4febe43cf.png)

2- Create PrivilagesRelationManager
![PrivilagesRelationManager](https://user-images.githubusercontent.com/16566638/169678166-ffd0a50b-9e58-42db-b8f0-3582dc8305ba.png)

3- Register PrivilagesManager at TeamR
![TeamResource](https://user-images.githubusercontent.com/16566638/169678177-aac33e71-db57-4914-9b8a-f563789c6d3c.png)
esource

4-Result (Nothing shown)
![Results](https://user-images.githubusercontent.com/16566638/169678192-b8068ca5-69eb-4c41-a810-06c7da0a9994.png)

**Comment by danharrin** [comment]
Do you have a PrivelagesPolicy? Make sure you can viewAny()

**Comment by ishaarany** [comment]
Yes, I have one, and have viewAny() method.

**Comment by danharrin** [comment]
Make sure it returns `true`

**Reply by ishaarany** [reply]
sure it returns true.
